app/lib/hooks/useSettings.tsx

```diff
@@ -5,19 +5,42 @@ import {
   isLocalModelsEnabled,
   LOCAL_PROVIDERS,
   providersStore,
+  useLatestBranch,
```

Comment:
only thing in this entire PR, should not be using  any variable name or function name that start with 'use'  its react convention to name hooks with this syntax. example 'useState', `useSettings`  etc, rename this to `latestBranch`

Reply:
fixed and thank you